src/redux/Dragons/DragonsSlice.js

```diff
@@ -0,0 +1,53 @@
+import { createSlice, createAsyncThunk, isRejectedWithValue } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
```

Comment:
I think got the URL missed up.  The one here is for fetching rockets, and not dragons.